src/pages/auth/Login.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import './login.css';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { clearError } from '../../states/slice/authSlice';
+import { authData } from '../../network/api-crud';
+import Sidebar from '../../components/sidebar/Sidebar';
+
+export default function Login() {
+
+    const [userName, setUserName] = useState('kminchelle');
+    const [password, setpassword] = useState('0lelplR');
+    const { data, error } = useSelector((state) => state.api);
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        clearErrorMessage()
+        if (!Array.isArray(data) && data !== undefined) {
```

Comment:
remove extra condition  data !== undefined as it is auto handle in !Array.isArray(data)

Reply:
done